packages/server/src/routes/plans/plans.test.ts

```
@@ -0,0 +1,154 @@
+			const response = await client["master-plans"].get();
+
+			expect(response.status).toBe(200);
+
+			const responseData = await response.json();
+			expect(responseData.data).toBeDefined();
+			expect(Array.isArray(responseData.data)).toBe(true);
+			expect(responseData.data.length).toBeGreaterThan(0);
+
+			// Check if our created plan is in the list
+			const createdPlan = responseData.data.find(
+				(plan: any) => plan.name === "Test Master Plan List",
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)